selfapi.js

```diff
@@ -304,7 +309,7 @@ API.prototype = {
 
         var expectedBody = null;
         if ('body' in exampleResponse) {
-          expectedBody = exampleResponse.body.trim();
+          expectedBody = jsonStringify(exampleResponse.body).trim();
```

Comment:
I like it 👍 Thanks for renaming this function! It's just a slight misnomer since it won't stringify if the input is already a string, but I think that using `maybeJsonStringify()` everywhere is too verbose. (Hm, `maybeStringify()`? But don't bother with this if it's not important to you, I'm fine with your code as it is now)